pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -222,13 +222,13 @@ def reset_test_dataloader(self, *args):
     def reset_val_dataloader(self, *args):
         """Warning: this is just empty shell for code implemented in other class."""
 
-    def _evaluate(self, model: LightningModule, dataloaders, max_batches: int, test_mode: bool = False):
+    def _evaluate(self, model: LightningModule, dataloaders, max_batches: List[int], test_mode: bool = False):
```

Comment:
```suggestion
    def _evaluate(self, model: LightningModule, dataloaders: List[DataLoader], max_batches: List[int], test_mode: bool = False):
```
let's complete the signature here while we're at it.
you will also have to import DataLoader